zio-flow-server/src/main/scala/zio/flow/server/Main.scala

```diff
@@ -0,0 +1,30 @@
+package zio.flow.server
+
+import zhttp.service.Server
+import zio._
+import zio.flow.ExecutionEnvironment
+import zio.flow.internal.KeyValueStore
+import zio.flow.serialization.{Deserializer, Serializer}
+import zio.flow.{ActivityError, ExecutionEnvironment, Operation, OperationExecutor}
+
+object Main extends ZIOAppDefault {
+  override def run: ZIO[Any with ZIOAppArgs with Scope, Any, Any] = (for {
+    endpoint <- ZFlowEndpoint.endpoint
+    _        <- Server.start(8090, endpoint)
+  } yield ())
+    .provide(
+      ZLayer.succeed(ExecutionEnvironment(Serializer.json, Deserializer.json)),
+      KeyValueStore.inMemory,
+      ZFlowService.layer,
+      ZFlowEndpoint.layer,
+      DurableLog.live,
+      IndexedStore.inMemory,
+      PersistentExecutor.make(
+        new OperationExecutor[Any] {
```

Comment:
We have now a real implementation of `OperationExecutor` but it seems like we miss a proper layer definition for it.

We will have to define a proper config descriptor that allows selecting different data stores and serialization formats etc. Will create a separate task for that.

Reply:
Have now defined and used `OperationExecutor.live` to inject the real implementation. 
But I'm not quite sure whether it's the idiomatic way to use `ZLayer.service` to pass the `OperationExecutor` into `PersistentExecutor.make`

Reply:
It is ok for now, the whole user facing API including creating the executors etc will be cleaned up before the first release